### PR TITLE
GH-864 Adds layout_CollapsibleList

### PR DESCRIPTION
**Pull request by jbrundage**
Fixes GH-864

@mlzummo @dtsnell4 @GordonSmith This was built with an emphasis on re-usability. Let me know if you think any changes might improve it.

https://rawgit.com/jbrundage/Visualization/CollapsibleList/demos/dermatology.html?src/layout/CollapsibleList

Signed-off-by: Jay Brundage <jaman.brundage@lexisnexis.com>

### CODE REVIEW

**Comment by jbrundage** [comment]
I'll rebase this onto 2.0.0 when it's available

**Comment by GordonSmith** [comment]
@jbrundage Milestones are like "due by" - if it makes it into an earlier version all the better

File: src/layout/CollapsibleList.js [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+                } else {
+                    return widget._target;
+                }
+                return li;
+            })
+        ;
+        rows.exit().remove();
+        rows.each(function(widget){
+            widget.render();
+        });
```
Comment (GordonSmith) [inline src/layout/CollapsibleList.js]: @jbrundage If your working off of latest sources I think lines 97->99 are not needed.

**Comment by jbrundage** [comment]
Then should I aim the PR at 1.8.0?

**Comment by GordonSmith** [comment]
Yes

**Pull request closed without merging**